src/style.css

```diff
@@ -0,0 +1,71 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
+body {
+  margin: 0;
+  font-family: Roboto;
+}
+
+header {
+
+  justify-content: space-between;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  flex-basis: 1024px;
+  flex-shrink: 0;
+  flex-grow: 1;
+  padding: 0 50px;
+}
+
+header,
+nav {
+  display: flex;
+}
+
+nav {
+  align-items: center;
+}
+
+header > img {
+  margin-top: 10px;
+  width: 40px;
+  height: 40px;
+}
+
+a {
+  position: relative;
+  margin-right: 20px;
+  font-family: Roboto;
+  font-size: 12px;
+  line-height: 15px;
+  color: #000;
+  text-decoration: none;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```

Comment:
where did you find such styles? 22.5, why not 22.3?

Reply:
https://prnt.sc/s559x0  here

Reply:
let is be as it is for now